src/main/scala/com/fulcrumgenomics/util/Rscript.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.util
+
+import java.nio.file.Path
+
+import dagr.commons.util.LazyLogging
+
+import scala.io.Source
+import scala.util.{Success, Try}
+
+/**
+  * Object that enables running of R scripts via the Rscript command line too.
+  */
+object Rscript extends LazyLogging {
+  /** The name of the Rscript executable. */
+  private val Executable = "Rscript"
```

Comment:
I think that's awkward.  That would imply that either every fgbio tool that uses R would need to take a parameter for where to find `Rscript`, or we'd have to have some global way of doing that.  Otherwise there's no way for a pipeline to tell the tool where to find Rscript.